app/dashboard/templates/shared/profile_rank_link.html

```diff
@@ -0,0 +1,23 @@
+{% comment %}
+  Copyright (C) 2019 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+    {% if is_my_profile %}
+      <a id="rank_link" href="javascript:show_persona_modal();">(Toggle)</a>
+    {% else %}
+      <a id="rank_link" href="{% url 'leaderboard' link %}">(View Leaderboard)</a>
+    {% endif %}
+
+
```

Comment:
reindent with 2 spaces

 remove extra lines

Reply:
is there an html linter you recommend?

Reply:
Ah i just set html + css + js to 2 spaces ! 
no good linter which works well with django template 😓